docs/components/radio.md

```diff
@@ -24,21 +24,21 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, reactive, toRefs } from 'vue'
-
-export default defineComponent({
-  setup() {
-    const data = reactive({
-      objA: { name: 'a' },
-      objB: { name: 'b' },
-      radio: true,
-    })
-
-    return {
-      ...toRefs(data),
-    }
-  }
-})
+  import { defineComponent, reactive, toRefs } from "vue";
```

Comment:
I think you may have an issue with your ESLint settings -- these shouldn't be indented

Reply:
Oops, stupid Prettier